http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/internal/NettyConfiguration.java

```diff
@@ -92,4 +89,12 @@ public boolean reapIdleConnections() {
     public TlsKeyManagersProvider tlsKeyManagersProvider() {
         return configuration.get(SdkHttpConfigurationOption.TLS_KEY_MANAGERS_PROVIDER);
     }
+
+    public TlsTrustManagersProvider tlsTrustManagersProvider() {
+        return configuration.get(SdkHttpConfigurationOption.TLS_TRUST_MANAGERS_PROVIDER);
+    }
+
+    public boolean trustAllCertificates() {
+        return configuration.get(TRUST_ALL_CERTIFICATES);
```

Comment:
Not directly related to your change, but can you make static vs not static import consistent?

Reply:
Done